mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -557,8 +598,33 @@
         <Arc value="0"/>
         <Range index="0" value="5"/>
     </CAbilEffectTarget>
+    <CAbilMorph id="LiberatorMorphtoAA">
+        <InfoArray index="0">
+            <SectionArray index="Abils">
+                <DurationArray index="Delay" value="0"/>
+            </SectionArray>
+            <SectionArray index="Mover">
+                <DurationArray index="Duration" value="0"/>
+            </SectionArray>
+            <SectionArray index="Stats">
+                <DurationArray index="Delay" value="0"/>
+            </SectionArray>
+        </InfoArray>
+    </CAbilMorph>
+    <CAbilMorph id="LiberatorMorphtoAG">
+        <InfoArray index="0">
+            <SectionArray index="Abils">
+                <DurationArray index="Delay" value="0"/>
+            </SectionArray>
+            <SectionArray index="Stats">
+                <DurationArray index="Duration" value="0"/>
+            </SectionArray>
+        </InfoArray>
+    </CAbilMorph>
     <CAbilEffectTarget id="LockOn">
         <Effect index="0" value="LockOnInitialSetNew"/>
+        <AutoCastValidatorArray value="CasterIsNotHidden"/>
```

Comment:
Fixed an issue where lock-on could enter cooldown while the Cyclone is loaded into a medivac

This validator prevents the auto-cast from activating when the Cyclone is inside a transport